pageserver/src/tenant/remote_timeline_client/download.rs

```diff
@@ -98,9 +90,11 @@ pub async fn download_layer_file<'a>(
 
             let bytes_amount = tokio::io::copy_buf(&mut reader, &mut destination_file)
                 .await
-                .with_context(|| format!(
+                .with_context(|| {
```

Comment:
This is an unrelated whitespace change that rustfmt has made as a side effect of removing the with_context() wrapper above.

Reply:
Yep, agree that there's still an issue for cancellation during the body receive -- would be good to tackle that in a separate PR.